server/index.js

```diff
@@ -1,8 +1,11 @@
 import express from 'express';
+import dotenv from 'dotenv';
+
+dotenv.config();
 
 const app = express();
 
-const port = 3000;
+const port = process.env.PORT || 3000;
 
 app.get('/', (req, res) => res.send('This is Hello Books'));
```

Comment:
The acceptance criteria stated that a wild card route to handle **any endpoint**.
_A wildcard route to handle any endpoints, to return a JSON message { "message": "Hello Books Deferral" }_
```suggestion
app.get('/*', (req, res) => res.send('Hello Books Deferral'));
```